src/ColorVectorSpace.jl

```diff
@@ -164,13 +171,15 @@ abs2(c::TransparentRGB) = (ret = abs2(color(c)); ret + convert(typeof(ret), alph
 norm(c::AbstractRGB) = sqrt(abs2(c))
 norm(c::TransparentRGB) = sqrt(abs2(c))
 
-one{C<:AbstractRGB}(::Type{C})     = C(1,1,1)
-one{C<:TransparentRGB}(::Type{C})  = C(1,1,1,1)
+oneunit{C<:AbstractRGB}(::Type{C})     = C(1,1,1)
```

Comment:
If you use `where`, you'll avoid deprecation warnings on 0.7.

Reply:
I want to save some work so that femocleaner feels valued. Even bots have feelings.